src/test/kotlin/racingcar/domain/result/CarRaceResultTest.kt

```diff
@@ -0,0 +1,39 @@
+package racingcar.domain.result
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import racingcar.domain.car.Car
+import racingcar.domain.car.vo.Name
+import racingcar.domain.move.MovingStrategy
+
+class CarRaceResultTest {
+    private val alwaysMovingStrategy = object : MovingStrategy {
+        override fun canMove(): Boolean {
+            return true
+        }
+    }
+
+    @Test
+    fun `우승자를 확인`() {
+        val 제이 = Car(Name("제이"))
+        val 제이슨 = Car(Name("제이슨"))
+        val 포비 = Car(Name("포비"))
+
+        포비.move(alwaysMovingStrategy)
+        val firstRound = CarRaceResultForRound(listOf(제이, 제이슨, 포비))
+
+        제이.move(alwaysMovingStrategy)
+        제이슨.move(alwaysMovingStrategy)
+        val secondRound = CarRaceResultForRound(listOf(제이, 제이슨, 포비))
+
+        제이.move(alwaysMovingStrategy)
+        val thirdRound = CarRaceResultForRound(listOf(제이, 제이슨, 포비))
+
+        val carRaceResult = CarRaceResult(listOf(firstRound, secondRound, thirdRound))
+
+        val winners = carRaceResult.findWinners()
+        assertThat(winners.size).isEqualTo(1)
+        assertThat(winners[0].name).isEqualTo(제이.name.value)
+        assertThat(winners[0].position).isEqualTo(제이.position.value)
```

Comment:
```suggestion
   @Test
    fun `우승자를 확인`() {
        val 제이 = Car(Name("제이"), Position(2))
        val 제이슨 = Car(Name("제이슨"), Position(1))
        val 포비 = Car(Name("포비"), Position(1))

        val carRaceResultForRound = CarRaceResultForRound(listOf(제이, 제이슨, 포비))
        val carRaceResult = CarRaceResult(listOf(carRaceResultForRound))
        val winners = carRaceResult.findWinners()
        
        assertAll({
            assertThat(winners.size).isEqualTo(1)
            assertThat(winners[0].name).isEqualTo(제이.name.value)
            assertThat(winners[0].position).isEqualTo(제이.position.value)
        })
    }
```

해당 테스트는 자동차 게임의 전체적인 테스트라고 생각해요!

자동차 결과에 대한 테스트에 대해서는 결과만 테스트해도 괜찮지 않을까요~?

재주님 의견을 듣고 싶네요!

Reply:
위에서 전체적인 테스트를 진행한 이유는,
선두가 뒤바뀌는 상황에서 마지막 라운드를 기준으로 우승자를 잘 추출하는지를 보기 위함이었는데,
마지막 상태를 기준으로 테스트해도 괜찮겠네요.

전체적인 테스트는 CarRaceResultTest에 있지 않아도 충분하겠네요 :)